cfgov/paying_for_college/disclosures/scripts/update_colleges.py

```diff
@@ -242,7 +242,9 @@ def update(exclude_ids=None, single_school=None, store_programs=False):
         set_school_grad_rate(school, data)
         compile_net_prices(school, data)
         program_data = api_utils.compile_school_programs(data)
-        if program_data and type(program_data.get("most_popular")) == list:
+        if program_data and isinstance(
+            type(program_data.get("most_popular")), list
+        ):
```

Comment:
I think you meant this instead? Doing `isinstance(type(...))` checks the type of the type, not the type of the element.

```suggestion
        if program_data and isinstance(program_data.get("most_popular"), list):
```